packages/block-library/src/cover/test/edit.js

```diff
@@ -375,4 +378,76 @@ describe( 'Cover block', () => {
 			} );
 		} );
 	} );
+
+	describe( 'isDark settings', () => {
+		test( 'should set is-light class if light background set', async () => {
+			const { container } = await setup();
+			const colorPicker = screen.getByRole( 'button', {
+				name: 'Color: White',
+			} );
+			await userEvent.click( colorPicker );
+
+			const coverBlock = // eslint-disable-next-line testing-library/no-node-access
+				container.getElementsByClassName( 'wp-block-cover' );
+
+			expect( coverBlock[ 0 ] ).toHaveClass( `is-light` );
```

Comment:
```suggestion
			await setup();
			const colorPicker = screen.getByRole( 'button', {
				name: 'Color: White',
			} );
			await userEvent.click( colorPicker );

			const coverBlock = screen.getByLabelText( 'Block: Cover' );

			expect( coverBlock ).toHaveClass( `is-light` );
```

Reply:
🤦 yep, no point in that classname selector, fixed.